src/main/java/com/yelp/nrtsearch/server/grpc/NrtsearchServer.java

```diff
@@ -158,6 +159,7 @@ public void start() throws IOException {
     GlobalState globalState = serverImpl.getGlobalState();
 
     registerMetrics(globalState);
+    initIngestionPlugin(globalState, plugins);
```

Comment:
I think the best place to do the initialization is https://github.com/Yelp/nrtsearch/blob/main/src/main/java/com/yelp/nrtsearch/server/grpc/NrtsearchServer.java#L387 after the global state is constructed. Doing this in `LuceneServerImpl` means it will work with the test server classes.

Reply:
done